src/main/java/com/modzy/sdk/JobClient.java

```diff
@@ -177,15 +177,17 @@ private void appendInput(Job job, String inputItemKey, String dataItemKey, Input
 				.path(job.getJobIdentifier()).path(inputItemKey).path(dataItemKey)
 				.request(MediaType.MULTIPART_FORM_DATA);
 		builder.header("Authorization", "ApiKey "+this.apiKey);
-		MultiPart data = new MultiPart();
+		BufferedInputStream bis;
+		MultiPart data;
 		byte inputBuffer[] = new byte[chunkSize];
 		int chunkByteCount;
 		try {
-			BufferedInputStream bis = new BufferedInputStream(inputValue, chunkSize);
+			bis = new BufferedInputStream(inputValue, chunkSize);
 			while( (chunkByteCount = bis.read(inputBuffer)) != -1 ){
-				logger.info("Adding input: "+job.getJobIdentifier()+" "+inputItemKey+" "+dataItemKey);
+				logger.info("Adding input: "+job.getJobIdentifier()+" "+inputItemKey+" "+dataItemKey+" ["+chunkByteCount+"]");
+				data = new MultiPart();
 				data.bodyPart( new StreamDataBodyPart("input", new ByteArrayInputStream(inputBuffer, 0, chunkByteCount), dataItemKey) );
-				Response response = builder.post(Entity.entity(data, data.getMediaType()));
+				Response response = builder.post(Entity.entity(data, data.getMediaType()));				
```

Comment:
Can you also please remove the extra spaces at the end so that this line doesn't appear changed in the commit?